hs_core/views/serializers.py

```diff
@@ -8,8 +8,10 @@
 from hs_core import hydroshare
 from .utils import validate_json, validate_user, validate_group
 from hs_access_control.models import PrivilegeCodes
+from drf_yasg.utils import swagger_serializer_method
 
 RESOURCE_TYPES = [rtype.__name__ for rtype in utils.get_resource_types()]
+CONTENT_TYPES = [ctype.__name__ for ctype in utils.get_content_types()]
 
```

Comment:
dunder name magic?  I just followed the pattern set for resource_types.  Unless there are clear limitations with this pattern, I'd prefer to keep instead of refactoring.